antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/misc/MurmurHash.kt

```diff
@@ -10,99 +10,90 @@ package org.antlr.v4.kotlinruntime.misc
  *
  * @author Sam Harwell
  */
-object MurmurHash {
+public object MurmurHash {
```

Comment:
What is a MurmurHash?

Reply:
See https://en.wikipedia.org/wiki/MurmurHash